selene_sdk/utils/performance_metrics.py

```diff
@@ -247,6 +247,30 @@ def get_feature_specific_scores(data, get_feature_from_index_fn):
     return feature_score_dict
 
 
+def auc_u_test(labels, predictions):
+
+    """ Outputs the area under the the ROC curve associated with a certain 
+    set of labels and the predictions given by the training model.
+
+    Parameters
+    ----------
+    labels: numpy.ndarray
+    Known labels of values predicted by model.
```

Comment:
Indent, specify the exact shape expected (i.e. is this 2 dimensional? what are the dimensions?)

Reply:
(add indents where needed in the rest of the params/return)